packages/config/config/devices/0x045a/Z-CM-V01.json

```diff
@@ -0,0 +1,218 @@
+{
+	"manufacturer": "ZVIDAR",
+	"manufacturerId": "0x045a",
+	"label": "Z-CM-V01",
+	"description": "Smart Curtain Motor",
+	"devices": [
+		{
+			"productType": "0x0904",
+			"productId": "0x0507"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Set To Start Holding Hands",
+			"description": "This parameter can be used to set the motor open hand start function",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Close hand start function",
+					"value": 0
+				},
+				{
+					"label": "Open hand start function",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Set Motor Direction",
+			"description": "This parameter can be used to set the motor rotation direction.",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Forward",
+					"value": 1
+				},
+				{
+					"label": "Opposite",
+					"value": 2
+				},
+				{
+					"label": "Reverse",
+					"value": 3
+				}
```

Comment:
What's the difference between those two?

Reply:
This is my interpretation:
1. forward is the "close" direction, and opposite is the "open" direction
3. is the reverse of 1. So "forward" is now the "open" direction, and the opposite is the "close" direction.

This is honestly a guess. I might factory reset one of these and play with settings to see what is what.

Reply:
Ok, I guess we can keep this as-is for now, but I'd like to clear this up at some point.